relay-dynamic-config/src/defaults.rs

```diff
@@ -476,6 +483,9 @@ fn span_metrics(transaction_extraction_enabled: bool) -> impl IntoIterator<Item
                 Tag::with_key("span.origin")
                     .from_field("span.origin")
                     .always(),
+                Tag::with_key("span.category")
+                    .from_field("span.sentry_tags.category")
+                    .always(), // already guarded by condition on metric
```

Comment:
The category and op are very similar (see snapshots). Do we need both?

Reply:
I need it for AI at least - `category = ai.pipeline` is how we identify the top level spans

Reply:
Does looking at the prefix of `op` work? When `category=ai.pipeline`, `op=ai.pipeline.whatever`. Cardinality should be the same, just wondering whether we need the extra tag.

Reply:
You can't do prefix matches like that in snuba from what I've read

Reply:
IMO having this tag is fine, it does not increase cardinality because it is derived from `op`.